manifest.json

```diff
@@ -0,0 +1,13 @@
+{
+    "name": "Casey's Extension",
+    "version": "1",
```

Comment:
Using semantic versioning (major.minor.bugfix/tweak, e.g. "8.4.2") is a good habit to get into. Major version bumps break compatibility with previous major versions. Minor version bumps add new features to the current major version without breaking backward compatibility. Bugfix/tweak version bumps fix issues with the existing features and modify/improve them.

Tl;dr: Go with "1.0.0" :)

Ts;wrm: https://semver.org/